src/AppKit/Enums.cs

```diff
@@ -2112,6 +2122,7 @@ public static class NSFileTypeForHFSTypeCode {
 	
 	// These constants specify the possible states of a drawer.
 	[Native]
+	[Deprecated (PlatformName.MacOSX, 10, 13, message: "Drawers are deprecated; consider using 'NSSplitViewController'")]
```

Comment:
also it should not say `deprecated` since the IDE already knowns from the attribute